exec/java-exec/src/main/java/org/apache/drill/exec/service/ServiceEngine.java

```diff
@@ -82,10 +89,32 @@ public Controller getController() {
     return controller;
   }
 
+  private void submit(ExecutorService p, final String name, final Closeable c) {
```

Comment:
Can we move this to AutoCloseables?

We can have the method be: AutoCloseables.closeQuietlyInParallel(AutoCloseable... closeables) or something similar?